stubs/ComponentModel/Container.stub

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Nette\ComponentModel;
+
+class Container extends Component implements IContainer
+{
+
+	/**
+	 * @template T of \Nette\ComponentModel\IComponent
+	 * @phpstan-param null|class-string<T> $filterType
+	 * @phpstan-return \Iterator<int|string, T>
```

Comment:
maybe we should add some condition return type here, if $filterType is null, it should be just `\Iterator<int|string, \Nette\ComponentModel\IComponent>`